contracts/token/ERC20.cairo

```diff
@@ -26,8 +25,19 @@ end
 func decimals() -> (res: felt):
 end
 
-@storage_var
-func initialized() -> (res: felt):
+#
+# Constructor
+#
+
+@constructor
+func constructor{
+        syscall_ptr : felt*, pedersen_ptr : HashBuiltin*,
+        range_check_ptr}(sender: felt):
```

Comment:
I think `recipient` would be a more appropriate name.

Reply:
Yes :)